cmd/tools/vdoc/tests/testdata/output_formats/main.html

```diff
@@ -1,54 +1,54 @@
 <section id="readme_main" class="doc-node">
 			<div class="title"><h1> main <a href="#readme_main">#</a></h1></div>
-<h2>Description</h2><p>This is an example of a an .md file, used for adding more rich text documentation in a project or module.</p><p>This is a <a href="https://vlang.io/">link</a> to the main V site.</p><p>This is a <b>bold text</b>.</p><p>This is a script <code>console.log('hi from README.md');</code> .</p><h2>Examples</h2><h3>Processing command line args</h3><pre><code class="language-v"><span class="token keyword">import</span> os
+<h2>Description</h2><p>This is an example of a an .md file, used for adding more rich textdocumentation in a project or module.</p><p>This is a <a href="https://vlang.io/">link</a> to the main V site.</p><p>This is a <b>bold text</b>.</p><p>This is a script <code>&lt;script&gt;console.log('hi from README.md');&lt;/script&gt;</code> .</p><h2>Examples</h2><h3>Processing command line args</h3><pre><code class="language-v"><span class="token keyword">import</span> os
```

Comment:
why is it `textdocumentation` here, but `text documentation` in the md file?

Reply:
is it because of merge_comments?

Reply:
Thanks for looking properly during the review @spytheman 

Likely not because of `merge_comments` since it is not used for the readme text. I think newlines need to be escaped, adding a space character.

Also, I should have kept `merge_comments_without_examples` here. This changed during testing yesterday and I forgot to revert it.

Gonna update and extend tests for both.